test/e2e/cluster_samples_operator_test.go

```diff
@@ -136,7 +136,7 @@ func verifyOperatorUp(t *testing.T) *samplesapi.Config {
 	return cfg
 }
 
-func verifyX86(t *testing.T) bool {
+func verifySupportedArch(t *testing.T) bool {
```

Comment:
If the e2e changes were a separate commit, it could be handled regardless of whose PR is merged.

Reply:
done